sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -254,468 +251,179 @@
     abstract void analyseSentimentForListLanguageHint();
 
     // Detect Language runner
-    void detectLanguageShowStatisticsRunner(BiConsumer<List<DetectLanguageInput>,
+    static void detectLanguageShowStatisticsRunner(BiConsumer<List<DetectLanguageInput>,
```

Comment:
What reason to make runner a static method?

Reply:
here, just to keep consistent with other methods in the file.

Reply:
We should make it non-static. I think make the method static will not be safe if we want the tests parallel.